src/CMakeLists.txt

```diff
@@ -28,6 +26,7 @@ add_library (task STATIC CLI2.cpp CLI2.h
                   rules.cpp
                   sort.cpp
                   util.cpp util.h)
+target_link_libraries(task taskchampion-cpp)
```

Comment:
This means that the include path for `taskchampion-cpp` will be used to compile the sources in `task`.